tox.ini

```diff
@@ -23,3 +25,12 @@ max-line-length = 100
 exclude = venv,.git,.tox,dist,*egg,cachito/web/migrations,.env
 # W504 line break after binary operator
 ignore = W504
+
+[testenv:integration]
+basepython = python3
+skipsdist = true
+commands =
+    pytest -rA -vvv \
+        --confcutdir=tests/integration \
+        {posargs:tests/integration}
+passenv = KRB5CCNAME
```

Comment:
What's this for?

Reply:
When using Kerberos authentication, tox needs this variable set (credential cache location) otherwise it will fail with error: kerberos.GSSError: ('No Kerberos credentials available: No KCM server found', -1765328243). But since there does not have to be Kerberos authentication used, I'll remove the variable from here and mention it in Readme, ok?

Reply:
+1 sounds like a good idea.